polygeist/tools/cgeist/Lib/clang-mlir.cc

```diff
@@ -301,6 +301,21 @@ void MLIRScanner::initSupportedFunctions() {
       "6access4modeE1025ELNS4_6targetE2014ELNS4_11placeholderE0ENS0_"
       "3ext6oneapi22accessor_property_listIJEEEE14getLinearIndexILi1EEEmNS0_"
       "2idIXT_EEEEUlmE_EEvSt16integer_sequenceImJXspT_EEEOT0_");
+
+  // Additional functions needed for parallel_for with one dimensional write
+  // buffer.
+
+  supportedFuncs.insert("_ZN4sycl3_V12idILi1EEC1ILi1ELb1EEERNSt9enable_ifIXeqT_"
+                        "Li1EEKNS0_4itemILi1EXT0_EEEE4typeE");
```

Comment:
Any chance to get constexpr/consteval mangler in clang?

Reply:
This list is meant to be removed when `cgeist` is tested better.